spyder/app/mainwindow.py

```diff
@@ -1522,6 +1522,19 @@ def load_window_settings(self, prefix, default=False, section='main'):
         return hexstate, window_size, prefs_dialog_size, pos, is_maximized, \
                is_fullscreen
 
+    def get_window_center(self, load=False):

```

Comment:
@dalthviz, I think you're assuming here that the window was maximized the previous time it was closed, which is normally the case. But what would happen if it was closed with half the size of the screen?

Reply:
It will appear centered relative to the last position of the MainWindow:

![autod](https://user-images.githubusercontent.com/16781833/68548987-2571ae00-03c1-11ea-891c-2a2ae1fb1dbf.gif)

Reply:
Mmmh, I don't like this too much. How is the splash screen correctly centered?

Reply:
The splash screen is always centered relative to the first screen (it doesn't depend on where the MainWindow will appear)

Reply:
Then let's do the same for the Autosave dialog. I think it's the best we can do.